NMS-16049: Update documentation (#32)
OpenNMS/helm-charts: docs/modules/installation/pages/installation.adoc

The install script requires a `.yaml` file that contains at least the domain name that you are using.
You may also want to specify the password for the OpenNMS instance.
+

> On GitHub, see https://github.com/OpenNMS/helm-charts/blob/main/opennms/values.yaml[values.yaml] for example configuration parameters you can specify.

Reviewer: ```suggestion
On GitHub, see https://github.com/OpenNMS/helm-charts/blob/main/opennms/values.yaml[values.yaml] for sample configuration parameters you can specify.
```

=== Start the OpenNMS environment on your Kubernetes cluster

NOTE: When running locally, you should run only one set.

The minimal command to load the Helm chart and start OpenNMS services assumes that the `values.yaml` file has been updated with your custom values.

[source, bash]
----
helm upgrade --install <namespace> ./horizon
----

> If you wish to customize your deployment, such as if you want to use one chart to deploy multiple instances into a cloud datacenter, you can call an additional yaml and/or include `--set` parameters to override values in the yaml file.

Reviewer: ```suggestion
If you want to customize your deployment (for example, to use one chart to deploy multiple instances into a cloud datacenter), you can call an additional yaml and/or include `--set` parameters to override values in the yaml file.
```

See the following topics might be useful after installation:

* xref:configuration:configuration.adoc[]
* xref:troubleshooting:debugging.adoc[]
* xref:reference:reference.adoc[]

> == Updating OpenNMS service settings

Reviewer: ```suggestion
== Update OpenNMS service settings
```

> If you make changes to the Helm chart definition, such as modifying `values.yaml`, updating one of the init scripts, or changing a container image version, you can redeploy the container by doing an upgrade of the helm chart.

Reviewer: ```suggestion
If you make changes to the Helm chart definition, such as modifying `values.yaml`, updating one of the init scripts, or changing a container image version, you can redeploy the container by upgrading the Helm chart.
```

Author: Tweaked the text to match closer to Helm's terminology.
```suggestion
If you make changes to the Helm chart definition, such as modifying `values.yaml`, updating one of the init scripts, or changing a container image version, you can redeploy the container by updating the Helm release.
```
